.github/workflows/ci.yml

```diff
@@ -370,11 +370,83 @@ jobs:
             dist/*.whl
             dist/*.tar.gz
           retention-days: 7
+          
+  test:
```

Comment:
I can see in line 340 there are some other unit tests going on. Hence, I am not sure if this is a duplicated job.

Pinging here @mainakk do provide more insights about it.

Reply:
Talked with @mkundu1 and resolved the issue, yea I accidently duplicated the tests. From 0.12 the pyfluent team separated the testing and building and while pulling from the latest ci_cd file I also added the separate test section along with the build_test. Should be fixed now with only one build_test job.